MediaBrowser.Api/Playback/MediaInfoService.cs

```diff
@@ -259,7 +259,7 @@ private async Task<PlaybackInfoResponse> GetPlaybackInfo(Guid id, Guid userId, s
                     // TODO handle supportedLiveMediaTypes ?
                     mediaSources = await _mediaSourceManager.GetPlayackMediaSources(item, user, true, false, CancellationToken.None).ConfigureAwait(false);
                 }
-                catch (Exception ex)
+                catch (Exception)
```

Comment:
Why not keep ex, log it and remove a TODO?

Reply:
oh right, I can do that

Reply:
There's no logger in there sigh.. I'd prefer to remove this change from this PR at this point instead, then I'll fix that todo in a new one.

Reply:
DI would be nice :) Up to you, I'm just reading through some PRs and comment where I think it makes sense

Reply:
you got it :) #656